packages/adapter-sveltekit/src/hooks/project-init.ts

```diff
@@ -224,6 +224,42 @@ const createPreviewRouteMatcherFile = async ({
 	});
 };
 
+const createPreviewAPIRoute = async ({
+	helpers,
+	options,
+}: SliceMachineContext<PluginOptions>) => {
+	const extension = await getJSFileExtension({ helpers, options });
+	const filename = path.join(
+		"src",
+		"routes",
+		"api",
+		"preview",
+		`+server.${extension}`,
+	);
+
+	if (await checkHasProjectFile({ filename, helpers })) {
+		return;
+	}
+
+	const contents = source`
+		import { redirectToPreviewURL } from '@prismicio/svelte/kit';
+		import { createClient } from '$lib/prismicio';
```

Comment:
Interesting about what $ will do here?

Reply:
SvelteKit comes with a built-in `$lib` path alias for `src/lib`. It is idiomatic to use the alias, and is helpful for us since we can generate files without computing relative paths.